metrics/metrics.go

```diff
@@ -599,6 +606,17 @@ var (
 		Measure:     RcmgrBlockMem,
 		Aggregation: view.Count(),
 	}
+
+	// hc
+	SubnetSpinUpDurationView = &view.View{
+		Measure:     SubnetSpinUpDuration,
+		Aggregation: defaultMillisecondsDistribution,
+		TagKeys:     []tag.Key{SubnetType},
+	}
+	SubnetCountView = &view.View{
```

Comment:
If we add this metric, we should ensure that we count the active subnets (we increase it when we register a new subnet, and we decrease when we leave it). Actually, I think it'll be even more interesting to track the number of subnet in each state `Active`, `Inactive` and `killed`.